test/unit_test/test_common.h

```diff
@@ -5,16 +5,31 @@
 
 #include <vector>
 
+#include "action_score.h"
+
 constexpr float FLOAT_TOL = 0.0001f;
```

Comment:
[Minor] Should this be a default value for `float_tolerance` on `check_collections_with_tol`?